.circleci/config.yml

```
@@ -547,6 +558,19 @@ workflows:
         requires:
         - setup_env_for_load_tests
 
+  run_soo_load_tests_on_stage:
+    triggers:
+      - schedule:
+          cron: "10 6 * * 1-5"
```

As SOO doesn't utilise CMS or DIR-API we can run those tests earlier
```suggestion
          cron: "40 4 * * 1-5"
```